webapp/sources/rudder/rudder-web/src/main/resources/rudder-system-global-parameter.conf

```diff
@@ -8,7 +8,8 @@
     },
     "server": {
       "cf_serverd_bind_address": "::"
-    }
+    },
+    "syslog_facility": "LOG_LOCAL3"
```

Comment:
```suggestion
    "agent_syslog_facility": "LOG_LOCAL3"
```

Reply:
IIRC it is only a setting for the agent

Reply:
if it's for agent only, it should be `"agent":  { "syslog_facility" : "LOG_LOCAL3" }`

If it's for both, it seems worthy to have a "log" category, it seems common enough to be useful. I'm not sure why but I would prefer to minimize number of top level props.

Reply:
it's for both cf-agent and cf-serverd

Reply:
so, for now we will put it in log, and we will try to split / make it more granular if we want/can in further releases:
```
{ ...
"log": { "syslog_facility" : "LOG_LOCAL3" }
...
}
```

Reply:
Pr updated